LSM303AGR.c

```diff
@@ -159,6 +161,88 @@ int32_t lsm303_acc_stop(void)
     return writeReg(LSM303_ACCEL, ACC_CTRL1, reg1);
 }
 
+int32_t lsm303_acc_setINT2(ACC_INT2_type_t mode, uint8_t threshold, uint8_t duration) //type, threshold, time
+{
+	int32_t err;        // error return for the function
+	uint8_t reg5;
+	uint8_t reg6;
+	uint8_t thres_reg;       // ACC threshold setting register 
+	uint8_t cfg_reg;       // ACC INT2 control register 
+	int j;
+	
+	switch(currentScale) {
+		case ACC_FS_2G:  j = 0; 
+		break;
+		case ACC_FS_4G:  j = 1;
+		break;
+		case ACC_FS_8G:  j = 2;
+		break;
+		case ACC_FS_16G: j = 3;
+		break;
+		default: j = 0;
+	};
+	
+	err = readReg(LSM303_ACCEL, ACC_CTRL5, &reg5);
+	err = readReg(LSM303_ACCEL, ACC_CTRL6, &reg6);
+	
+	reg6 |= ACC_CTRL6_I2_INT2;
+	thres_reg = ((threshold) >> j); 
+	cfg_reg = (ACC_INT2_6DMOVE_XYZ); 
```

Comment:
if this is always a constant it should not be a variable.
why is it always the same? we will ALWAYS get X, Y and Z... High AND low?